src/test/clone/CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ target_link_libraries(test_clone ${GLIB_LIBRARIES}
 ## register the tests
 add_test(NAME clone COMMAND test_clone)
 
-add_test(NAME clone-shadow-ptrace COMMAND
-    ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d
-    clone.shadow_ptrace.data ${CMAKE_CURRENT_SOURCE_DIR}/clone_test_config.xml)
+add_test(
+	NAME clone-shadow-ptrace
+	COMMAND sh -c "\
+        ${yaml2xml} ${CMAKE_CURRENT_SOURCE_DIR}/clone_test_config.yaml --output - \
+	| ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d clone.shadow_ptrace.data - \
+	"
+)
```

Comment:
A lot of the tests are mixing spaces and tabs for indentation, could you modify the indentation to be consistent within each test, and preferably between tests? (Ideally 4 spaces for indentation, but being consistent is the most important.)

For example, this test might be formatted like:

```
add_test(
    NAME clone-shadow-ptrace
    COMMAND sh -c "\
    ${yaml2xml} ${CMAKE_CURRENT_SOURCE_DIR}/clone_test_config.yaml --output - \
    | ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d clone.shadow_ptrace.data - \
    "
)
```

Reply:
arff good catch !

Reply:
A configuration error with my IDE. I updated and its looks good now